tdrs-frontend/cypress.config.js

```diff
@@ -9,6 +9,7 @@ module.exports = defineConfig({
 
     env: {
       apiUrl: 'http://localhost:8080/v1',
+      adminUrl: 'http://localhost:8080/admin',
```

Comment:
Nit: suggest to use env vars instead of hardcoding the urls.

Reply:
These are env vars which can be overriden in `package.json`. Is that sufficient or do you think they should be accessible from something like `process.env.adminUrl`?